js/components/meta-select-control.js

```diff
@@ -24,24 +25,32 @@ const MetaSelectControl = compose(
 			metaKey: 'sub_menu',
 			metaValue: 0,
 		};
+
+		let primaryMenus;
+		apiFetch( { path: '/hello-admin/v1/menus' } ).then( ( menus ) => {
+			primaryMenus = menus;
+		} );
+
```

Comment:
This isn't returning a value because I believe the $menu global is not set before this api hook is called. I wanted to leave this in however, so you can see what I was trying to achieve.